tests/test_events.py

```diff
@@ -36,46 +36,6 @@ def test_status_assignment(event_scraper,
             assert event.status == scraper_assigned_status
 
 
-@pytest.mark.parametrize('item_sequence,should_error', [
-    (12, True),
-    (11, False),
-])
-def test_sequence_duplicate_error(event_scraper,
```

Comment:
What about testing that the (mock) alert is sent, rather than removing this test?